docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -350,6 +350,282 @@ and a file system location of ``/var/my_tables/test_table``::
 
 .. _iceberg-materialized-views:
 
+Special tables
+--------------
+
+The Iceberg connector makes available several hidden tables that can provide insights regarding
```

Comment:
The Iceberg connector maintains several hidden tables that provide metadata for a specific table. You can query each metadata table by appending the metadata table name to the table name::

   SELECT * FROM "test_table$data"